plugins/BEdita/API/src/Controller/ResourcesController.php

```diff
@@ -97,15 +94,126 @@ protected function findAssociation($relationship)
     {
         $relationship = Inflector::underscore($relationship);
         if (array_key_exists($relationship, $this->getConfig('allowedAssociations'))) {
-            $associations = TableRegistry::get($this->modelClass)->associations();
+            $associations = $this->Table->associations();
             foreach ($associations as $association) {
                 if ($association->property() === $relationship) {
                     return $association;
                 }
             }
         }
 
-        throw new NotFoundException(__('Relationship "{0}" does not exist', $relationship));
+        throw new NotFoundException(__d('bedita', 'Relationship "{0}" does not exist', $relationship));
+    }
+
+    /**
+     * Paginated list of resources.
```

Comment:
`index()` now is used also for entity creation
I like having simplified routing rules, but It seems to me that `index()` and `view()` do too many different things - breaking Single Responsibility Principle
What about a having `add()` `edit()` `delete()` like before, with routing method?

@batopa what do you think?